source/fixtures.rst

```diff
@@ -28,7 +30,7 @@ using the function
    CONTAINER_WITH_HEALTHCHECK = DerivedContainer(
        base="registry.opensuse.org/opensuse/leap:latest",
        containerfile="HEALTHCHECK CMD true",
-       healthcheck_timeout_ms=None,
+       healthcheck_timeout=timedelta(seconds=-1),
```

Comment:
Should that not be the following?

```suggestion
       healthcheck_timeout=None,
```

Reply:
Resolved in offline discussion. Opposed to `timedelta(seconds=-1)`, `None` sets the healthcheck to auto. -1 disables it, which is the desired state here.